### PR TITLE
DOC: Frontend documentation

### PR DESCRIPTION
**Pull request by Integraluminium**
Readme for developers to help them setup the project

### CODE REVIEW

**Review by Jozys — CHANGES_REQUESTED** [review]
Two little issues that needs to be addressed, than this PR can be merged.

File: frontend/src/components/CircularProgress.tsx [hunk 1]
```diff
@@ -1,3 +1,7 @@
+/**
+ * Renders animated circular progress that goes from 0 deg (0%) to 180 deg (100%) and renders the percentage in between
```
[the remark this answers is not in the document]
Comment (benniloidl) [reply frontend/src/components/CircularProgress.tsx:2]: No because it is only a half circle.

[the remark this answers is not in the document]
Comment (Jozys) [reply frontend/src/components/CircularProgress.tsx:2]: Okay, than everything here is fine.

**Review by benniloidl — APPROVED** [review]

**Review by Jozys — APPROVED** [review]
Looks good now.

**Pull request merged**